packages/calcite-components/src/components/list/list.tsx

```diff
@@ -189,7 +189,11 @@ export class List
   @Prop({ mutable: true }) selectedItems: HTMLCalciteListItemElement[] = [];
 
   /**
-   * Specifies the selection mode - `"multiple"` (allow any number of selected items), `"single"` (allow one selected item), `"single-persist"` (allow one selected item and prevent de-selection), or `"none"` (no selected items).
+   * Specifies the selection mode of the component:
+   * `"multiple"` allow any number of selected items,
+   * `"single"` allow only one selected item,
+   * `"single-persist"` allow one selected item and prevent de-selection,
+   * `"none"` does not allow any selections.
```

Comment:
Saw this looks similar to your recent PR, but it differs from some of the other components, such as `accordion` and `table`. WDYT is the best approach for the `selectionMode` prop across components? Maybe this PR should include an update across all components for consistency based on what we think works the best comprehensively.

Reply:
I was thinking about this as well actually, was going to probably make a separate PR for it but I can include it here since its essentially the same kind of changes.

As for best practice, I looked into it and there's not really anything definitive, I think what we have is probably fine and we just need to better align the wording across the different components since they should pretty much be the same. 

One thing that was bothering me is the way these types of descriptions looked, the way I see it in code wasn't reflected in how it ends up being formatted on the site. I'm talking about each value being on a new line which to me makes it way easier to read. 

So there's two ways to actually get it to make a new line. One way is to add two spaces and the end of each line and the other way is to separate the lines by an empty line. I think the two spaces looks cleaner but its easy to not realize that they are even present in the code. 

The two spaces at the end of each line would look identical to what you see now but here's what the other way looks like:
![Screenshot 2024-01-04 at 9 54 04 AM](https://github.com/Esri/calcite-design-system/assets/60022782/22af59a5-58ca-4cd4-8970-82b75556c76c)
(⬆️ I think this way is probably the one to go with IMO)


Both of them results in the description looking like this:![image](https://github.com/Esri/calcite-design-system/assets/60022782/6df2b300-7c7d-45f7-81ef-d091467bdad0)

Instead of what they look like now: ![image](https://github.com/Esri/calcite-design-system/assets/60022782/5512247c-a64f-43d3-8fb1-745dad4433f6)

Now I don't know if there will be a change between what we see and how it ends up on the doc site but assuming it's the same formatting, I think the individual line one looks better but LMKWYT.

Reply:
Also I was thinking they would actually look like this: ![Screenshot 2024-01-04 at 10 12 40 AM](https://github.com/Esri/calcite-design-system/assets/60022782/fce6ac1d-2b4c-45fa-abbc-85fa45863165)

Where each value has a ":" and has it's description as a standalone sentence.

Reply:
> I was thinking about this as well actually, was going to probably make a separate PR for it but I can include it here since its essentially the same kind of changes.

Awesome! 
✨ 💪🏻✨  



> One thing that was bothering me is the way these types of descriptions looked, the way I see it in code wasn't reflected in how it ends up being formatted on the site. I'm talking about each value being on a new line which to me makes it way easier to read.

Agreed 💯

> Now I don't know if there will be a change between what we see and how it ends up on the doc site but assuming it's the same formatting, I think the individual line one looks better but LMKWYT.

There may be a few changes, albeit minor, but here are some examples for context from [`accordion`](https://developers.arcgis.com/calcite-design-system/components/accordion/), [`combobox`](https://developers.arcgis.com/calcite-design-system/components/combobox), and [`list`](https://developers.arcgis.com/calcite-design-system/components/list/):

![image](https://github.com/Esri/calcite-design-system/assets/5023024/a4948a3e-a3f5-4961-8cf3-9a1c06507fcb)
![image](https://github.com/Esri/calcite-design-system/assets/5023024/6351d432-ad56-4c3f-a7bf-abe476ecac38)
![image](https://github.com/Esri/calcite-design-system/assets/5023024/9ff07788-fc00-4bc1-9e0c-356e2a65bf98)

There are a few other properties throughout the design system that have a similar structure as well, so we may want to also change those depending on the change, such as removing the parenthesis `"("` and `")"` characters, such as [note: not a comprehensive list]:
- `block-section`'s `toggleDisplay`
- `combobox`'s `selectionDisplay`
- `list`'s `selectionAppearance`
- `shell-panel`'s `displayMode`

> Also I was thinking they would actually look like this: ![Screenshot 2024-01-04 at 10 12 40 AM](https://private-user-images.githubusercontent.com/60022782/294264358-fce6ac1d-2b4c-45fa-abbc-85fa45863165.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ0ODIzMTQsIm5iZiI6MTcwNDQ4MjAxNCwicGF0aCI6Ii82MDAyMjc4Mi8yOTQyNjQzNTgtZmNlNmFjMWQtMmI0Yy00NWZhLWFiYmMtODVmYTQ1ODYzMTY1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA1VDE5MTMzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY4YTJjYmM5MDk0ZDZkNGY4ODQ2MTdmMGYyZmEyMDM5YjQ4ODI5Y2M0ZWVmYWM0ZjAwN2ZjZDFiMjE5ODdiMjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.0gD4zjnPrXdU32YAldS9c_UbQzrwjADSkGx7HWYvAI8)
> 
> Where each value has a ":" and has it's description as a standalone sentence.

While that might be the case in the repo/typings, the doc site would look a bit different, and would defer to keeping the doc site look and feel consistent over the repo to ensure consistency for our users there, which is usually the first place folks navigate to.


> The two spaces at the end of each line would look identical to what you see now but here's what the other way looks like:
> ![Screenshot 2024-01-04 at 9 54 04 AM](https://private-user-images.githubusercontent.com/60022782/294259973-22af59a5-58ca-4cd4-8970-82b75556c76c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ0ODIzMTQsIm5iZiI6MTcwNDQ4MjAxNCwicGF0aCI6Ii82MDAyMjc4Mi8yOTQyNTk5NzMtMjJhZjU5YTUtNThjYS00Y2Q0LTg5NzAtODJiNzU1NTZjNzZjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA1VDE5MTMzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk5YjE4NWI4OGQ4NDM0NDAyZjdmYjQ5Y2Y1NmI1ZTgwODg3NDA4YmRiN2U4NDNjNWM5YTQ1M2VhZTM4YzRlMWImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.X4Y4QSJ_eybwHLaIPo-zfmKfoFva59cDfmTpFG3ez7A)
> (⬆️ I think this way is probably the one to go with IMO)

With the above in mind, your proposal above looks fantastic. 🌟 It would be easier for folks to read, and understand within the repo/typings and on the doc site where the parenthesis aren't present. 🏆

You could open a new PR to accommodate the additional properties with parenthesis mentioned above (and additional ones identified), or use this existing PR for the new consistency - whichever you prefer.

With the above changes, the PR has changed a bit, WDYT about changing the title to something more generic like `(docs) consistent api description refs` or similar?

Reply:
I think another PR for the additional properties makes sense to keep this one from getting too big.